app/views/unattended/provisioning_templates/PXELinux/coreos_pxelinux.erb

```diff
@@ -4,7 +4,11 @@ name: CoreOS PXELinux
 model: ProvisioningTemplate
 oses:
 - CoreOS
-%>
+description: |
+  The template to render PXELinux bootloader configuration for CoreOS Container Linux distribution.
+  This does not work for Fedora CoreOS or Flatcar Container Linux, which use Ignition instead
+  The output is deployed on the host's subnet TFTP proxy.
+-%>
```

Comment:
Since CoreOS Container Linux is EOL shall we drop a note here? Just thinking, perhaps not. Maybe to remove it later (Deprecation).

Reply:
I keep the list I want to propose for dropping later after this is in, so far:

* coreos (replaced by fedora/flatcar),
* poap,
* RancherOS PXELinux (replaced by k3os),
* WAIK (likely replaced by other windows templates), 
* Atomic Kickstart default (is there still support for RH Atomic?), 
* CoreOS provision, 
* rancher os and coreos snippets (content of their provisioning templates), 
* http_proxy - unused snippet

but I'll open proper discourse thread for that